docs/getting-started.md

```diff
@@ -0,0 +1,629 @@
+```javascript
+@Consumer
+class MovieDetail extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      movie:  {}
+    }
+    this.fetch = this.fetch.bind(this)
+    this.fetch()
+  }
+
+  fetch() {
+    const { movie } = this.state
+    const { imdbID, caption, show_poster } = this.props
+    this.fetchContent({
+      movie: {
+        source: 'movie-find',
+        query: { "imdbID": imdbID },
+        transform: (data) => {
+          const movieData = data || {}
+
+          if (!show_poster) {
+            movieData.Poster = null
+          }
+          movieData.caption = caption
+          //this.state.movie = movie
+          return movieData
+        }
+      }
+    })
+    this.render()
+  }
+
+  render () {
+
+    const { Actors, Director, Plot, Poster, Rated, Title, Writer, Year, caption } = this.state.movie || {}
+
+    return (
+      <div className='movie-detail col-sm-12 col-md-8'>
+        <div className="card" style={{"flexDirection": "row", display: "flex"}}>
+          {Poster && Title && <div className="card-img-top"><img src={Poster} alt={`Poster for ${Title}`} /></div>}
+
+          <div
+            className="card-body"
+            style={{ margin: "0 10px" }}
+            >
+            {Title && <h1>{Title}</h1>}
+            {Year && <p><strong>Year:</strong> {Year}</p>}
+            {caption && <blockquote>"{caption}"</blockquote>}
+
+          </div>
+        </div>
+      </div>
+    )
+  }
+}
+
+MovieDetail.label = 'Movie Detail'
+
+export default MovieDetail
+
+```
+
+A few things have changed here. Our component now takes `imdbID`, `caption` and `show_poster` as props. It no longer uses the global content source. The `caption` and `show_poster` fields are factored into the render function. But most importantly, `imdbID` is used as an argument to our content source. This is pretty cool!
+
+
+We're not quite done, though. We'll need to tell our feature pack that custom embeds are a valid content element to appear in an article body. So let's find our article body feature (the location and code varies by project) and add this to our content element switch statement:
+
+```javascript
+                        case 'custom_embed':
+                          return <CustomEmbedBody element={element} />
+```
+
+We'll also need to import the component at the top, perhaps like:
+
+```
+import CustomEmbedBody from './_children/custom-embed'
+```
+
+And we'll need to implement the custom embed component as well.
+
+```javascript
+'use strict'
+
+/* Third party libs */
+import React, { Component } from 'react'
+
+/* Components */
+import MovieDetail from '../../../movies/movie-detail'
+
+/* Other JS */
+
+/* Non-JS resources */
+
+
+class CustomEmbedBody extends Component {
+
+  constructor(props) {
+    super(props)
+
+    this.element = this.props.element
+  }
+
+  render() {
+
+    return (
+      <div className="customEmbed">
+        <MovieDetail
+          imdbID={this.element.embed.id}
```

Comment:
Same, just use `this.props.element` wherever you need to. You can destructure it into a shorter variable at the top of the function if you'd like.